app/src/main/java/com/asfoundation/wallet/ui/settings/entry/SettingsFragment.kt

```diff
@@ -81,12 +84,8 @@ class SettingsFragment : PreferenceFragmentCompat(), SettingsView {
     presenter.onResume()
   }
 
-  override fun onDestroyView() {
-    presenter.stop()
-    super.onDestroyView()
-  }
-
   override fun onDestroy() {
+    presenter.stop()
```

Comment:
Is there any reason why this was moved from onDestroyView to here? I think it should probably be in onDestroyView